src/raftstore/store/store.rs

```diff
@@ -1196,13 +1199,16 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         self.region_ranges.insert(enc_end_key(&region), region.get_id());
     }
 
-    fn on_ready_result(&mut self, region_id: u64, exec_results: Vec<ExecResult>) {
+    fn on_ready_result(&mut self,
+                       region_id: u64,
+                       exec_results: Vec<ExecResult>,
+                       raftlog_gc_tasks: &mut RaftlogGcTasks) {
         // handle executing committed log results
```

Comment:
I think we can handle task here directly, no need pass the arg and handle outside.